website/docs/guides/features-block.html.markdown

```diff
@@ -240,6 +241,8 @@ The `template_deployment` block supports the following:
 
 The `virtual_machine` block supports the following:
 
+* `detach_implicit_data_disk_on_deletion` - (Optional) Should the `azurerm_virtual_machine_implicit_data_disk_from_source` resource detach the implicit data disk of the Virtual Machine when the `azurerm_virtual_machine_implicit_data_disk_from_source` resource is destroyed? Defaults to `false`.
```

Comment:
```suggestion
* `detach_implicit_data_disk_on_deletion` - (Optional) Should we detach the `azurerm_virtual_machine_implicit_data_disk_from_source` from the virtual machine instead of destroying it? Defaults to `false`.
```